ui/app/AppLayouts/Chat/views/CreateChatView.qml

```diff
@@ -33,7 +33,9 @@ Page {
         delegate: Item {
             property string publicId: model.pubKey
             property string name: model.name
+            property string nickName: model.nickName
```

Comment:
isn't it `localNickname`?

Reply:
it's ok since in the `roleNames` he exposed `localNickName` as `nickName`

Reply:
I can change to localNickName in all places to avoid misunderstanding